astro/src/content/docs/extend/code/lambdas/jwt-populate.mdx

```diff
@@ -23,22 +24,26 @@ This lambda is not executed if you call the Login API directly and omit the `app
 If you are using the API to create the lambda you will need to ensure your function has the following signature:
 
 ```javascript
-function populate(jwt, user, registration) {
+function populate(jwt) {
   // Lambda code goes here
 }
 ```
 
-This lambda must contain a function named `populate` that takes three parameters. The parameters that the lambda is passed are:
+This lambda must contain a function named `populate` that takes up to 4 parameters. The parameters that the lambda is passed are:
 
 * `jwt` - the claims object to be signed and return as the JWT payload. You can modify this object.
 * `user` - the FusionAuth User object. This object is read-only.
 * `registration` - the FusionAuth UserRegistration object. This parameter will be `undefined` when the user is not registered for the application. This object is read-only.
+* `context` - <AvailableSince since="1.99.9"/> - an object containing the context of the request. This object is read-only.
+
+Context includes the following:
+* `scopes` - <AvailableSince since="1.99.9"/> - array of scopes that were requested for the token.
 
 The two FusionAuth objects are well documented in the [User API](/docs/apis/users) and [Registration API](/docs/apis/registrations) documentation. The JWT object is a JavaScript object containing the JWT payload. See [OpenID Connect & OAuth 2.0 Token](/docs/lifecycle/authenticate-users/oauth/tokens).
 
 You may add or modify anything in the `jwt` object. However, you may not modify the header keys or values of the JWT. FusionAuth also protects certain reserved claims. The following claims are considered reserved and modifications or removal will not be reflected in the final JWT payload:
 
-- `exp`
+- `exp` - prior to version `1.99.9` the the `exp` claim was considered reserved and could not be removed or modified. Since version `1.99.9`, the `exp` claim can be modified but the value must be less than or equal to the original `exp` value.
```

Comment:
kinda think all of this might look better in a table, your thoughts?

Reply:
I think with only 2 cases this is very readable / understandable

Reply:
I found a grammar error Andy missed! :)